### ISSUE TITLE
 Chronograf (1.8.8) Expected number or array but got string:  CPULoad cpu chronograf-v1-e001d2b6-c6c7-4ceb-b515-2f0bbec9af8d nil map[cpu:cpu4 host:telegraf-getting-started]  CRITICAL map[value:5.75692963752564] 5.75692963752564 2021-03-09 23:30:40  0000 UTC .

### ISSUE THREAD

**Issue by ieasydevops** [influxdata/chronograf#5692]
What browser are you using (name and version)?

What operating system are you using?

Please describe what you were trying to do when you encountered this error:

```
Error: Expected number or array but got string:  CPULoad cpu chronograf-v1-e001d2b6-c6c7-4ceb-b515-2f0bbec9af8d nil map[cpu:cpu4 host:telegraf-getting-started]  CRITICAL map[value:5.75692963752564] 5.75692963752564 2021-03-09 23:30:40  0000 UTC .
    at j (http://localhost:8888/src.2f700edb.js:3164:1898)
    at W.parseArray_ (http://localhost:8888/src.2f700edb.js:3164:31911)
    at W.start_ (http://localhost:8888/src.2f700edb.js:3164:35574)
    at W.updateOptions (http://localhost:8888/src.2f700edb.js:3164:36541)
    at p.value (http://localhost:8888/src.2f700edb.js:3224:5554)
    at no (http://localhost:8888/src.2f700edb.js:717:84207)
    at ao (http://localhost:8888/src.2f700edb.js:717:85725)
    at http://localhost:8888/src.2f700edb.js:717:98848
    at Object.exports.unstable_runWithPriority (http://localhost:8888/src.2f700edb.js:713:3303)
    at Xo (http://localhost:8888/src.2f700edb.js:717:98783)
```

**Comment by sranka** [comment]
Can you pls describe the context of where and how it happens? It seems that it is a data-driven issue, a string value is returned where a number is expected.

**Comment by sranka** [comment]
Closing it was fixed in 1.8.9 with https://github.com/influxdata/chronograf/pull/5613
